src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,103 @@
+package task;
+
+import java.util.ArrayList;
+
+import ui.Ui;
+/**
+ * Class <code>TaskList</code> represents a list of <code>Task</code>object using <code>ArrayList</code>.
+ * Contains <code>addTask</code> and <code>deleteTask</code> which can add/delete tasks in the list.
+ */
+public class TaskList {
+
+    protected ArrayList<Task> tasklist;
+
```

Comment:
Maybe you can consider setting this variable to "private" for encapsulation? Also, there are no classes inheriting from TaskList that would require access to this variable, so it should not be protected?

Perhaps you can also rename tasklist to tasks? I think it is more natural to have TaskList having tasks rather than TaskList having tasklist.